### PR TITLE
CUMULUS-3737: Prepare release 18.3.1

### PR DESCRIPTION
**Pull request by paulpilone**
**Summary:** Summary of changes

Addresses [CUMULUS-3737: Release 18.3.1](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3737)

## Changes

* Reverts CUMULUS-3449
* Adds CUMULUS-3779
* Updates CL and fixes position of previously unreleased CUMULUS-2891

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Comment by paulpilone** [comment]
This PR should be very similar to the https://github.com/nasa/cumulus/pull/3686 PR plus https://github.com/nasa/cumulus/pull/3689.

**Comment by Jkovarik** [comment]
For posterity - this should be similar to https://github.com/nasa/cumulus/pull/3686 and https://github.com/nasa/cumulus/pull/3689

File: CHANGELOG.md [hunk 1]
```diff
@@ -186,16 +181,17 @@ operations (e.g. `PREFIX-AsyncOperationEcsLogs`).
   - Minor refactor of `@cumulus/lzards-api-client` to:
     - Use proper ECMAScript import for `@cumulus/launchpad-auth`
     - Update incorrect docstring
-- **CUMULUS-3449**
-  - Updated `@cumulus/db` package and configure knex hook postProcessResponse to convert the return string
-    from columns ending with "cumulus_id" to number.
+    - Use proper ECMAScript import for `@cumulus/launchpad-auth`
```
Comment (Jkovarik) [inline CHANGELOG.md:184]: This looks like it was accidentally duplicated

File: CHANGELOG.md [hunk 2]
```diff
@@ -62,6 +56,11 @@ output or status of your request. If you want to directly observe the progress
 of the migration as it runs, you can view the CloudWatch logs for your async
 operations (e.g. `PREFIX-AsyncOperationEcsLogs`).
 
+#### CUMULUS-3779 async_operations Docker image version upgrade
+  
+The `async_operations` Docker image has been updated to support Node v20 and `aws-sdk` v3. Users will need to bump
+the version tag of `async_operations` to at least 52 if using the Docker image.
```
Comment (Jkovarik) [inline CHANGELOG.md:62]: Is this published to Dockerhub?   We should make direct reference to the project/tag.

**Review by Jkovarik — APPROVED** [review]
Approved for merge, with the caveat that the CHANGELOG update should be addressed/discussed. 

To review I validated the changes appear to be a proper inclusion of the changes in the prior PRs listed that were reviewed/approved.

File: CHANGELOG.md (hunk 2 above)
Comment (paulpilone) [reply CHANGELOG.md]: Updated with a link directly to the project and tag. I also included a section for the `cumulus-ecs-task` image with a direct link to that version.

**Pull request merged**